src/ethereum_test_tools/vm/opcode.py

```diff
@@ -205,171 +205,4898 @@ class Opcodes(Opcode, Enum):
-    PUSH32 = Opcode(0x7F, pushed_stack_items=1, data_portion_length=32)
-
-    DUP1 = Opcode(0x80, pushed_stack_items=1, min_stack_height=1)
-    DUP2 = Opcode(0x81, pushed_stack_items=1, min_stack_height=2)
-    DUP3 = Opcode(0x82, pushed_stack_items=1, min_stack_height=3)
-    DUP4 = Opcode(0x83, pushed_stack_items=1, min_stack_height=4)
-    DUP5 = Opcode(0x84, pushed_stack_items=1, min_stack_height=5)
-    DUP6 = Opcode(0x85, pushed_stack_items=1, min_stack_height=6)
-    DUP7 = Opcode(0x86, pushed_stack_items=1, min_stack_height=7)
-    DUP8 = Opcode(0x87, pushed_stack_items=1, min_stack_height=8)
-    DUP9 = Opcode(0x88, pushed_stack_items=1, min_stack_height=9)
-    DUP10 = Opcode(0x89, pushed_stack_items=1, min_stack_height=10)
-    DUP11 = Opcode(0x8A, pushed_stack_items=1, min_stack_height=11)
-    DUP12 = Opcode(0x8B, pushed_stack_items=1, min_stack_height=12)
-    DUP13 = Opcode(0x8C, pushed_stack_items=1, min_stack_height=13)
-    DUP14 = Opcode(0x8D, pushed_stack_items=1, min_stack_height=14)
-    DUP15 = Opcode(0x8E, pushed_stack_items=1, min_stack_height=15)
-    DUP16 = Opcode(0x8F, pushed_stack_items=1, min_stack_height=16)
-
-    SWAP1 = Opcode(0x90, min_stack_height=2)
-    SWAP2 = Opcode(0x91, min_stack_height=3)
-    SWAP3 = Opcode(0x92, min_stack_height=4)
-    SWAP4 = Opcode(0x93, min_stack_height=5)
-    SWAP5 = Opcode(0x94, min_stack_height=6)
-    SWAP6 = Opcode(0x95, min_stack_height=7)
-    SWAP7 = Opcode(0x96, min_stack_height=8)
-    SWAP8 = Opcode(0x97, min_stack_height=9)
-    SWAP9 = Opcode(0x98, min_stack_height=10)
-    SWAP10 = Opcode(0x99, min_stack_height=11)
-    SWAP11 = Opcode(0x9A, min_stack_height=12)
-    SWAP12 = Opcode(0x9B, min_stack_height=13)
-    SWAP13 = Opcode(0x9C, min_stack_height=14)
-    SWAP14 = Opcode(0x9D, min_stack_height=15)
-    SWAP15 = Opcode(0x9E, min_stack_height=16)
-    SWAP16 = Opcode(0x9F, min_stack_height=17)
-
-    LOG0 = Opcode(0xA0, popped_stack_items=2)
-    LOG1 = Opcode(0xA1, popped_stack_items=3)
-    LOG2 = Opcode(0xA2, popped_stack_items=4)
-    LOG3 = Opcode(0xA3, popped_stack_items=5)
-    LOG4 = Opcode(0xA4, popped_stack_items=6)
-
-    RJUMP = Opcode(0xE0, data_portion_length=2)
-    RJUMPI = Opcode(0xE1, popped_stack_items=1, data_portion_length=2)
-    RJUMPV = Opcode(0xE2)
-
-    CREATE = Opcode(0xF0, popped_stack_items=3, pushed_stack_items=1)
-    CALL = Opcode(0xF1, popped_stack_items=7, pushed_stack_items=1)
-    CALLCODE = Opcode(0xF2, popped_stack_items=7, pushed_stack_items=1)
-    RETURN = Opcode(0xF3, popped_stack_items=2)
-    DELEGATECALL = Opcode(0xF4, popped_stack_items=6, pushed_stack_items=1)
-    CREATE2 = Opcode(0xF5, popped_stack_items=4, pushed_stack_items=1)
-
-    STATICCALL = Opcode(0xFA, popped_stack_items=6, pushed_stack_items=1)
-
-    REVERT = Opcode(0xFD, popped_stack_items=2)
-    INVALID = Opcode(0xFE)
-
-    SELFDESTRUCT = Opcode(0xFF, popped_stack_items=1)
-    SENDALL = Opcode(0xFF, popped_stack_items=1)
+STOP = Opcode(0x00)
```

Comment:
Indentation is missing in this and all subsequent lines.
```suggestion
    STOP = Opcode(0x00)
```